django_extensions/admin/__init__.py

```diff
@@ -15,12 +15,12 @@
 from django.utils.encoding import smart_str
 from django.utils.translation import ugettext as _
 from django.utils.text import get_text_list
-from django.contrib.admin import ModelAdmin
+from django.contribimport import admin
```

Comment:
This break tests https://travis-ci.org/django-extensions/django-extensions/builds/173107246

Maybe is `from django.contrib import admin`